packages/reports/addon/components/navi-column-config.js

```diff
@@ -214,7 +219,10 @@ class NaviColumnConfig extends Component {
   removeColumn(column) {
     const { type, fragment } = column;
     const removalHandler = this[`onRemove${capitalize(type)}`];
-    removalHandler?.(fragment);
+    if (removalHandler) {
+      set(column, 'isRemoving', true);
+      later(() => removalHandler(fragment), 200);
```

Comment:
I think we might want to just directly call this. Just playing around with it makes removing columns feel slow